docs/errors.md

```diff
@@ -0,0 +1,14 @@
+## Inline Functions cannot have resource requirements
```

Comment:
This file should be a reference for error messages. Please add the error message in a formatted box as the first thing after the title.

If a standardized and specific error message is not produced, then this entry should be moved to the FAQ.